data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+	outfits
+		"Phased Array Laser (Blue)" 6
+		"Phased Array Laser (Green)" 3
+		"Dipole Ramscoop"
+		
+		"Stellar Fusion Core"
+		"Radiant Fusion Core"
+		"Photoelectric Scales"
+		"Ultracapacitor Bank" 6
+		"Specular Deflector" 2
+		"Optical Lasing Generator" 4
+		"Multispectral Scanner" 9
+		"Electrostatic Ramscoop" 3
+		"Concealment Scales" 3
+		"Nanoceramic Scales"
+		"Radiative Scales"
+		"Electromotive Rifle" 600
+		
+		"Beam Core Antimatter Torch"
+		"Hyperdrive"
+
+	turret 36 183 "Phased Array Laser (Blue)"
+	turret -36 183 "Phased Array Laser (Blue)"
+	turret -44 354 "Phased Array Laser (Blue)"
+	turret 44 354 "Phased Array Laser (Blue)"
+	turret -28.5 -333 "Phased Array Laser (Blue)"
+	turret 28.5 -333 "Phased Array Laser (Blue)"
+	turret 0 6 "Phased Array Laser (Green)"
+	turret 0 -367 "Phased Array Laser (Green)"
+	turret 0 480 "Phased Array Laser (Green)"
+	gun 0 -384 "Dipole Ramscoop"
+
+	bay "Drone" -18 420
+	bay "Drone" 18 420
+	bay "Drone" -18 432
+	bay "Drone" 18 432
+	bay "Drone" -18 447
+	bay "Drone" 18 447
+	bay "Drone" -18 459
+	bay "Drone" 18 459
+	bay "Fighter" -21 432
+	bay "Fighter" 21 432
+	bay "Fighter" -21 420
+	bay "Fighter" 21 420
+	bay "Fighter" -21 447
+	bay "Fighter" 21 447
+	bay "Fighter" -21 459
+	bay "Fighter" 21 459
+
+	engine -215 -475
+		angle 9
+		zoom 2
+	engine 215 -475
+		angle -9
+		zoom 2
+	engine -186 -450
+		angle 9
+		zoom 2
+	engine 186 -450
+		angle -9
+		zoom 2
+
+	"steering engine" -5 498
+		angle 90
+		zoom 0.1
+		left
+	"steering engine" 5 498
+		angle -90
+		zoom 0.1
+		right
+	"steering engine" -35 -318
+		angle 90
+		zoom 0.1
+		right
+	"steering engine" 35 -318
+		angle -90
+		zoom 0.1
+		left
+
+	leak "leak" 60 15
+	leak "flame" 60 15
+	explode "tiny explosion" 15
+	explode "small explosion" 12
+	explode "medium explosion" 9
+	explode "large explosion" 9
+	explode "huge explosion" 9
+	explode "nuke explosion" 3
+	"final explode" "nuke explosion" 3
+	"final explode" "nuke residue fast" 90
+
+	description "This was once an Avgi sublight starship, one of the last models built before the discovery of the hyperdrive. Most starships around this period used ram augmented fusion drives to reach cruising speeds of up to 0.25c, but the Tachytia class was designed to reach a velocity of 0.4c with an antimatter fuel source to ignite propellant gathered with its huge ramscoop."
+	description "This particular ship was never launched on its mission, commissioned just before the hyperdrive made it obsolete. Now, it has been extensively modified by removing many of its propellant tanks, replacing them with a hyperdrive, additional equipment spaces, and several docking bays. The cryosleep bay has also been substituted for spacious habitatation cylinders, making this a true city ship."
+
+
+
+# Interceptors
+
+ship "Tremoros"
+	plural "Tremoros"
+	sprite "ship/avgi tremoros"
+	thumbnail "thumbnail/avgi tremoros"
+	attributes
+		category "Interceptor"
+		licenses
+			"Avgi Civilian"
+		"cost" 490000
```

Comment:
```suggestion
		"cost" 780000
```
This is larger, tougher and better-armed than any human interceptor, so a higher hull cost makes sense (For comparison, the Hawk is the most expensive human interceptor, with a hull cost of 670,000).